wp-content/plugins/core/src/Theme/Theme_Subscriber.php

```diff
@@ -29,7 +29,7 @@ private function media(): void {
 		$this->image_wrap();
 		$this->image_links();
 		$this->disable_responsive_images();
-		$this->oembed();
+		//$this->oembed();
```

Comment:
@jbrinley , just want to point out that I have disabled our custom oembed filter for the time being so default embed can work.